ibm/service/vpc/resource_ibm_is_instance_template.go

```diff
@@ -239,6 +242,14 @@ func ResourceIBMISInstanceTemplate() *schema.Resource {
 										ForceNew:    true,
 										Description: "The CRN of the [Key Protect Root Key](https://cloud.ibm.com/docs/key-protect?topic=key-protect-getting-started-tutorial) or [Hyper Protect Crypto Service Root Key](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) for this resource.",
 									},
+									isInstanceTemplateVolAttTags: {
+										Type:        schema.TypeSet,
+										Optional:    true,
+										ForceNew:    true,
```

Comment:
why forcenew to true we don't have update support?

Reply:
No harini, other fields in the same block has the same structure. 
https://pages.github.ibm.com/vpc/vpc-spec-artifacts/branch/master/public/2022-03-29.html#/Instances/update_instance_template

```
						isInstanceTemplateVolAttVolPrototype: {
							Type:     schema.TypeList,
							MaxItems: 1,
							MinItems: 1,
							Optional: true,
							ForceNew: true,
							Elem: &schema.Resource{
								Schema: map[string]*schema.Schema{
									isInstanceTemplateVolAttVolIops: {
										Type:        schema.TypeInt,
										Optional:    true,
										ForceNew:    true,
										Description: "The maximum I/O operations per second (IOPS) for the volume.",
									},
.
.
..
.
.

									isInstanceTemplateVolAttTags: {
										Type:        schema.TypeSet,
										Optional:    true,
										ForceNew:    true,
										Elem:        &schema.Schema{Type: schema.TypeString, ValidateFunc: validate.InvokeValidator("ibm_is_volume", "tags")},
										Set:         flex.ResourceIBMVPCHash,
										Description: "UserTags for the volume instance",
									},
								},
							},
						},
```